looper/conductor.py

```diff
@@ -767,6 +772,31 @@ def _reset_curr_skips(self):
         self._curr_skip_pool = []
         self._curr_skip_size = 0
 
+    def check_executable_path(self, pl_iface):
+        """Determines if supplied pipelines are callable.
+        Raises error and exits Looper if not callable"""
+        pl_iface = pl_iface
+        pipeline_commands = []
+        if "path" in pl_iface.keys():
+            pipeline_commands.append(pl_iface["path"])
+        if (
+            "var_templates" in pl_iface.keys()
+            and "pipeline" in pl_iface["var_templates"].keys()
+        ):
+            pipeline_commands.append(pl_iface["var_templates"]["pipeline"])
+        for command in pipeline_commands:
+            try:
+                result = is_command_callable(command)
+            except:
+                _LOGGER.error(f" {command} IS NOT EXECUTABLE. EXITING")
```

Comment:
I think you can add error message to `SampleFailedException`

Reply:
I decided to leave this one as is for now.